run-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Build docker (can comment out if already built)
+docker-compose build
+# Test the frontend

```

Comment:
This is great!! 

I find it a little hard to read in the console, so I recommend adding some helpful print statements.
Here's an idea that uses colour, screenshot following.
```
# Constants to add colour to text
BLUE='\033[0;34m'
NC='\033[0m' # No Color

# Test the backened
printf "**** ${BLUE}Begin backend tests${NC} ***\n"
docker-compose run \
  --rm backend ./manage.py test
printf "**** Complete backend tests ***\n"

# Test the frontend
printf "**** ${BLUE}Begin frontend tests${NC} ***\n"
docker-compose run \
  --rm frontend npm test
printf "**** Complete frontend tests ***\n"
```

![image](https://user-images.githubusercontent.com/4640747/51341670-dd635280-1a5f-11e9-9f4f-801677e53be5.png)

Reply:
I can add this easily enough

Reply:
Added in latest commit